functional-tests/src/main/java/com/redhat/che/selenium/core/client/RhCheTestWorkspaceServiceClient.java

```diff
@@ -40,7 +40,7 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(RhCheTestWorkspaceServiceClient.class);
   private static final String CREATE_WORKSPACE_REQUEST_JSON_PATH =
-      "/configs/create-workspace-request.json";
+      "/requests/workspace/osio/che-starter_vertx.json";
```

Comment:
Is there a reason why you chose `/requests` path instead of `/templates/workspace` as is used in upstream? Wouldn't make sense to put json templates for using with che-starter into `/templates/workspace/che-starter` or something like that? Currently I'm not entirely sure which way I prefer, but I'm slightly leaning towards `/templates/...` side.

Reply:
I've put it into `/requests` because it's effectively a request template for che-starter. this is not contacting the master API directly

Reply:
Ok, I can move the location to `/templates/che-starter` rather than `/templates/workspace/che-starter` because the `workspace` directory contains a `JSON` version of the `YAML` DC. The che-starter templates just don't fit there.

Reply:
files in `/templates/workspace/<infra>` are just workspace configs... json files describing workspaces to be created... I think, that adding `che-starter` into the workspace folder, it is more fitting, as it contains json files describing what workspace should be cerated ;-)

Reply:
:+1: